lib/rautomation/adapter/helper.rb

```diff
@@ -31,7 +31,8 @@ def build_solution(ext, platform)
         name = File.basename(ext, File.extname(ext))
         cmd = "msbuild /p:Configuration=Release ext\\#{name}\\#{name}.sln"
         cmd += " && #{cmd} /p:Platform=x64" if platform =~ /x64/ && name != 'WindowsForms'
-        system(cmd)
+        system(cmd) or raise StandardError, "An error occurred when trying to build solution #{name}. " +
+                                            "Make sure msbuild binary is in your PATH and the project is configured correctly"
```

Comment:
`...the project is configured correctly` - what does it mean? If anyone would get this error, then this error is not that helpful at all. Shouldn't the project be configured correctly already by the gem maintainer and not when compiling? 

Stating that `msbuild` should be in the path is an actionable point and might be helpful on the other hand.